src/main/resources/TestNGStandaloneTest.ftl

```diff
@@ -0,0 +1,14 @@
+package ${package};
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.jboss.arquillian.testng.Arquillian;
+
+
+public class ${ClassToTest}Test extends Arquillian {
+
+    @Test
+    public void should_be_deployed() {
+        Assert.assertNotNull(url);
```

Comment:
True , good catch, it was there because I was thinking in one use cases that works in cube and algeron which injects URL in standalone but this does not happens in Core.